phoenicis-javafx/pom.xml

```diff
@@ -143,6 +143,16 @@
                     <mainClass>org.phoenicis.javafx.JavaFXApplication</mainClass>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.zenjava</groupId>
```

Comment:
Don't you want to package this like here: https://github.com/PhoenicisOrg/phoenicis/pull/1389/files ?

Reply:
This is basically what is done here, except that I am not using.a snapshot version of the plugin (the downside is that it won't work with Java 9+)

Reply:
Yes

Reply:
Ok.